src/utils/transform-filters.ts

```diff
@@ -0,0 +1,74 @@
+import { CombinedFilter, Filter, FilterCombinator } from '@yext/answers-core';
+import { SelectableFilter } from '../models/utils/selectablefilter';
+
+/**
+ * Combine a given list of Filter using OR condition into a single CombinedFilter object
```

Comment:
nit: I have a suggestion for wording: "Combine a list of Filters using the logical OR operator into a CombinedFilter"